third_party/move/tools/move-formatter/tests/formatter/use/input1.move.fmt

```diff
@@ -0,0 +1,11 @@
+/// test_point: A list consisting of multiple items, with comments after the items
+
+module test_use {
+    use aptos_framework::aptos_coin::AptosCoin;
```

Comment:
Please add a test case where the use statement, because of importing multiple items, overflows the 90 column limit. This is actually quite a likely scenario.

Reply:
we added a new test case:
tests/formatter/use/input6.move